### ISSUE TITLE
OpenAudioMC support?

### ISSUE THREAD

**Issue by Blaixen** [DRE2N/DungeonsXL#1026]
**The feature you'd like to know more about**

Does this plugin have OpenAudioMC support?

I want to know because I want to add music to my dungeons, but whenever I put OpenAudioMC speakers in the dungeon and then test it, the music doesn't play.
**Link to the wiki article (in case it exists)**
There is no article for this.
**What did you already try?**
I tried to Use the speaker function from openaudiomc. and now I'm wondering is there support for it?
**Relevant configuration files**

(The three grave accents mark the beginning and end of a code block. If there are relevant configuration files,
please paste them in the lines between the accents so that others can see if there are any syntax errors.)

```
there are no relevant config files.
```

**Comment by Sataniel98** [comment]
No, plugins that save data per world are not supported. Might be an addition for the future.
